srcpkgs/electrum/template

```diff
@@ -1,25 +1,27 @@
 # Template file for 'electrum'
 pkgname=electrum
-version=3.3.8
-revision=2
+version=4.0.9
+revision=1
 build_style=python3-module
-pycompile_module="electrum electrum_gui electrum_plugins"
-hostmakedepends="python3-PyQt5-devel-tools python3-setuptools"
-depends="python3-aiohttp python3-aiohttp_socks python3-aiorpcx>=0.18<0.19
- python3-dnspython python3-ecdsa python3-jsonrpclib python3-protobuf
- python3-pyaes python3-pycryptodome python3-PyQt5 python3-qrcode"
-# Missing optional dependencies:
+hostmakedepends="python3-setuptools python3-PyQt5-devel-tools"
+depends="python3-aiohttp>=3.3.0<4.0.0 python3-aiohttp_socks>=0.3
+ python3-aiorpcx>=0.18<0.19 python3-bitstring python3-dnspython python3-ecdsa
```

Comment:
I don't like the version restrictions here, and would prefer to leave them unspecified.

Reply:
Done!